instrumentation/elastic-opentelemetry-instrumentation-openai/examples/README.md

```diff
@@ -0,0 +1,71 @@
+# OpenAI Zero-Code Instrumentation Examples
+
+This is an example of how to instrument OpenAI calls with zero code changes,
+using `opentelemetry-instrument` included in the Elastic Distribution of
+OpenTelemetry Python ([EDOT Python][edot-python]).
+
+When OpenAI examples run, they export traces, metrics and logs to an OTLP
+compatible endpoint. Traces and metrics include details such as the model used
+and the duration of the LLM request. In the case of chat, Logs capture the
+request and the generated response. The combination of these provide a
+comprehensive view of the performance and behavior of your OpenAI usage.
+
+## Install
+
+First, set up a Python virtual environment like this:
+```bash
+python3 -m venv .venv
+source .venv/bin/activate
+pip install -r requirements.txt
+```
+
+Next, install [EDOT Python][edot-python] and dotenv which is a portable way to
+load environment variables.
+```bash
+pip install "python-dotenv[cli]" elastic-opentelemetry
+```
+
+Finally, run `edot-bootstrap` which analyzes the code to add relevant
+instrumentation, to record traces, metrics and logs.
+```bash
+edot-bootstrap --action=install
+```
+
+## Configure
+
+Minimally, update the [.env](.env) file with your `OPENAI_API_KEY`.
+
+An OTLP compatible endpoint should be listening for traces, metrics and logs on
+`http://localhost:4317`. If not, update `OTEL_EXPORTER_OTLP_ENDPOINT` as well.
+
+## Run
+
+There are two examples, and they run the same way:
+
+### Chat
+
+[chat.py](chat.py) asks the LLM a geography question and prints the response.
+
+Run it like this:
+```bash
+dotenv run -- opentelemetry-instrument python chat.py
```

Comment:
I think we are missing the .env file (ack it is usually not checked in by default, so easy to miss)

Reply:
Eh, I have it in .gitignore :sweat:

Reply:
Pushed updated README that asks to copy `default.env` to `.env`